payments/todopago/__init__.py

```diff
@@ -0,0 +1,147 @@
+from decimal import Decimal
+from uuid import uuid4
+
+from django.http import HttpRequest
+from django.shortcuts import redirect
+from python_todopago import TodoPagoConnector
+from python_todopago.helpers import Item
+
+from payments import PaymentError
+from payments import PaymentStatus
+from payments import RedirectNeeded
+from payments.core import BasicProvider
+from payments.models import BasePayment
+
+STATUS_MAP = {
+    "APROBADA": PaymentStatus.CONFIRMED,
+}
+
+
+class TodoPagoProvider(BasicProvider):
+    """This backend implements payments using `TodoPago <https://todopago.com.ar/>`_.
+    You'll need to install with extra dependencies to use this::
+
+        pip install "django-payments[todopago]"
+
+    :param token: The token provided by TP
+    :param merchant: The merchant ID provided by TP
+    :param sandbox: Whether to use sandbox more.
+    """
+
+    def __init__(self, token: str, merchant: str, sandbox: bool):
+        self.client = TodoPagoConnector(token, merchant, sandbox=sandbox)
+        self.sandbox = sandbox
+
+    def authorize_operation(self, payment: BasePayment):
```

Comment:
Can you add return types to the methods for this class?

Reply:
Is this enough? https://github.com/jazzband/django-payments/pull/292/commits/cec96b999369cf3a4d79d7d1ab853646dbb209a2